tests/unit/NetworkTest.js

```diff
@@ -175,36 +161,34 @@ test('consecutive API calls eventually succeed when authToken is expired', () =>
                     authToken: 'qwerty12345',
                 }))
 
-                // Get&returnValueList=personalDetailsList
-                .mockImplementationOnce(() => Promise.resolve({
-                    jsonCode: 200,
-                    personalDetailsList: TEST_PERSONAL_DETAILS,
-                }))
+                .mockImplementation((command, data) => {
+                    const response = {jsonCode: 200};
 
-                // Get&returnValueList=account
-                .mockImplementationOnce(() => Promise.resolve({
-                    jsonCode: 200,
-                    account: TEST_ACCOUNT_DATA,
-                }))
```

Comment:
The order of the requests changed

You can see the original code expected request to be resolved in reverse order 
1. TEST_PERSONAL_DETAILS
2. TEST_ACCOUNT_DATA
3. TEST_CHAT_LIST

While the actual `API.GET` calls are
1. TEST_CHAT_LIST
2. TEST_PERSONAL_DETAILS
3. TEST_ACCOUNT_DATA

When we resume connectivity order of the requests shouldn't matter - they are quickly sent in parallel anyway
If for some reason we want to test the order then better to happen in a separate test

BTW the order now is preserved to match the `API.GET` original call order (second list above)